File: conference/registration-bc/web/src/main/java/com/microsoft/conference/registration/domain/Orders/Events/OrderReservationConfirmed.cs
using System;

namespace Registration.Orders
{
    public class OrderReservationConfirmed : OrderEvent
    {
        public OrderStatus OrderStatus { get; private set; }

        public OrderReservationConfirmed() { }
        public OrderReservationConfirmed(Guid conferenceId, OrderStatus orderStatus)
            : base(conferenceId)
        {
            OrderStatus = orderStatus;
        }
    }
}

File: samples/conference/registration-bc/src/main/java/com/microsoft/conference/registration/domain/orders/Events/OrderReservationConfirmed.java
package com.microsoft.conference.registration.domain.orders.Events;

import com.microsoft.conference.registration.domain.orders.Models.OrderStatus;

public class OrderReservationConfirmed extends OrderEvent {
    public OrderStatus orderStatus;

    public OrderReservationConfirmed() {
    }

    public OrderReservationConfirmed(String conferenceId, OrderStatus orderStatus) {
        super(conferenceId);
        this.orderStatus = orderStatus;
    }
}
